library/Magicbane/App.hs

```diff
@@ -1,5 +1,5 @@
 {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
-{-# LANGUAGE NoImplicitPrelude, NoMonomorphismRestriction, OverloadedStrings, UnicodeSyntax, DataKinds, TypeOperators, MultiParamTypeClasses, TypeFamilies, FlexibleContexts, FlexibleInstances, UndecidableInstances, GeneralizedNewtypeDeriving, CPP #-}
```

Comment:
Don't remove `OverloadedStrings` anywhere. Even if there are no strings in the file, I think it should be enabled *everywhere*.

Reply:
👍

Reply:
This should all be addressed now. I split out the concurrency stuff to `Magicbane.Async` for clarity of intent, and updated the readme to list `lifted-async` among the included packages.